assemble/bin/accumulo-cluster

```diff
@@ -210,7 +210,7 @@ function start_all() {
 
   for queue in $COMPACTION_QUEUES; do
     Q="COMPACTOR_HOSTS_${queue}"
-    for compactor in "${!Q}"; do
+    for compactor in ${!Q}; do
```

Comment:
I made this change because scan servers were not starting. I had two groups of three scan servers.  When I debugged it all three scan servers in a group would be consumed in one pass of the for loop.  When start_service got all three at once it silently did nothing.  Removing the quotes around `${!Q}` caused the for loop to go 3 times as expected.